### PR TITLE
Contribute WindowBuilder 1.15 M1 for Eclipse 2024-03

### PR DESCRIPTION
**Pull request by ptziegler**

### CODE REVIEW

File: windowbuilder.aggrcon [hunk 1]
```diff
@@ -34,12 +34,6 @@
     <features name="org.eclipse.wb.swing.doc.user.feature.feature.group">
       <categories href="simrel.aggr#//@customCategories[identifier='General%20Purpose%20Tools']"/>
     </features>
-    <features name="org.eclipse.wb.xwt.feature.feature.group" enabled="false">
```
Comment (jonahgraham) [inline windowbuilder.aggrcon:37]: @ptziegler or @merks can you have a look at this PR. I think it introduced errors in the aggregator, perhaps due to missing the change to the simrel.aggr file that went along with removing the features.

image

**Comment by ptziegler** [comment]
>  perhaps due to missing the change to the simrel.aggr file that went along with removing the features.

That seems to be the reason. My apologies, I'll try to get this fixed once I'm back from work.

**Comment by merks** [comment]
It was trivial to fix and is already done.  Don't worry, be happy.

**Comment by ptziegler** [comment]
Thanks!

**Pull request merged**